File: Data Structures & Algorithms/rotate-array/submission-0.java
class Solution {
    public void rotate(int[] nums, int k) {
        int n=nums.length;
        int ans[]=new int[nums.length];
        for(int i=0;i<nums.length;i++){
            ans[(i+k)%n]=nums[i];
        }
        for(int i=0;i<n;i++){
            nums[i]=ans[i];
        }
    }
}

File: Data Structures & Algorithms/two-integer-sum/submission-0.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        indexed_nums = sorted([(n, i) for i, n in enumerate(nums)])
        left, right = 0, len(indexed_nums) - 1
        result = []
        while left < right:
            current_sum = indexed_nums[left][0] + indexed_nums[right][0]
            if current_sum == target:
                result = sorted([indexed_nums[left][1], indexed_nums[right][1]])
                break
            elif current_sum < target:
                left += 1
            else:
                right -= 1
        return result